src/type/textCore.js

```diff
@@ -53,40 +53,466 @@ function textCore(p5, fn) {
+   * }
+   * </code>
+   * </div>
+   *
+   * <div>
+   * <code>
+   * function setup() {
+   *   textSize(32);
+   *   fill(255);
+   *   stroke(0);
+   *   strokeWeight(4);
+   *   text('hi', 50, 50);
+   *
+   *   describe('The text "hi" written in white with a black outline.');
+   * }
+   * </code>
+   * </div>
+   *
+   * <div>
+   * <code>
+   * function setup() {
+   *   background('black');
+   *   textSize(22);
+   *   fill('yellow');
+   *   text('rainbows', 6, 20);
+   *   fill('cornflowerblue');
+   *   text('rainbows', 6, 45);
+   *   fill('tomato');
+   *   text('rainbows', 6, 70);
+   *   fill('limegreen');
+   *   text('rainbows', 6, 95);
+   *
+   *   describe('The text "rainbows" written on several lines, each in a different color.');
+   * }
+   * </code>
+   * </div>
+   *
+   * <div>
+   * <code>
+   * function setup() {
+   *   background(200);
+   *   let s = 'The quick brown fox jumps over the lazy dog.';
+   *   text(s, 10, 10, 70, 80);
+   *
+   *   describe('The sample text "The quick brown fox..." written in black across several lines.');
+   * }
+   * </code>
+   * </div>
+   *
+   * <div>
+   * <code>
+   * function setup() {
+   *   background(200);
+   *   rectMode(CENTER);
+   *   let s = 'The quick brown fox jumps over the lazy dog.';
+   *   text(s, 50, 50, 70, 80);
+   *
+   *   describe('The sample text "The quick brown fox..." written in black across several lines.');
+   * }
+   * </code>
+   * </div>
+   *
+   * <div modernizr='webgl'>
+   * <code>
+   * let font;
+   *
+   * function preload() {
+   *   font = loadFont('assets/inconsolata.otf');
+   * }
+   *
+   * function setup() {
+   *   createCanvas(100, 100, WEBGL);
+   *   textFont(font);
+   *   textSize(32);
+   *   textAlign(CENTER, CENTER);
+   * }
+   *
+   * function draw() {
+   *   background(0);
+   *   rotateY(frameCount / 30);
+   *   text('p5*js', 0, 0);
+   *
+   *   describe('The text "p5*js" written in white and spinning in 3D.');
+   * }
+   * </code>
+   * </div>
    */
 
   /**
-   * TODO
+   * Sets the way text is aligned when <a href="#/p5/text">text()</a> is called.
+   *
+   * By default, calling `text('hi', 10, 20)` places the bottom-left corner of
+   * the text's bounding box at (10, 20).
+   *
+   * The first parameter, `horizAlign`, changes the way
+   * <a href="#/p5/text">text()</a> interprets x-coordinates. By default, the
+   * x-coordinate sets the left edge of the bounding box. `textAlign()` accepts
+   * the following values for `horizAlign`: `LEFT`, `CENTER`, or `RIGHT`.
+   *
+   * The second parameter, `vertAlign`, is optional. It changes the way
+   * <a href="#/p5/text">text()</a> interprets y-coordinates. By default, the
+   * y-coordinate sets the bottom edge of the bounding box. `textAlign()`
+   * accepts the following values for `vertAlign`: `TOP`, `BOTTOM`, `CENTER`,
+   * or `BASELINE`.
    *
    * @method textAlign
    * @for p5
+   * @param {Constant} horizAlign horizontal alignment, either LEFT,
```

Comment:
We can put the constants directly in the type now in 2.0:
```suggestion
   * @param {LEFT|CENTER|RIGHT} horizAlign horizontal alignment
```